README.md

```diff
@@ -8,6 +8,10 @@ Contains:
 
 * The Import Global Response Set (GRS) Tool for importing and updating response sets from a spreadsheet
 
+* The Export Users Tool to export a list of all users and their groups from iAuditor into a spreadsheet
+
+* The Sync Users Tool to import a list of users from a spreadsheet into iAuditor and assign them to groups
```

Comment:
The Sync Users Tool allows you to use a spreadsheet to keep your users and their group associations up to date and automatically reflect that in iAuditor.